app/src/main/AndroidManifest.xml

```diff
@@ -65,6 +65,12 @@
         android:name="android.permission.INTERACT_ACROSS_USERS_FULL"
         tools:ignore="ProtectedPermissions" />
 
+    <!-- This is needed to read if the IGNORE_ALLOW_BACKUP_IN_D2D -->
```

Comment:
Better comment needed. Wasn't sure what to write. It is needed to call `CompatChanges.isChangeEnabled(IGNORE_ALLOW_BACKUP_IN_D2D, packageName, UserHandle.of(userId)`

Reply:
Perhaps:

"This is needed to read check if apps are eligible for d2d"